chaoslib/litmus/disk_fill/disk_fill_by_litmus.yml

```diff
@@ -61,17 +61,20 @@
 
     - name: Patch the engine UID
       template:
-        src:  /chaoslib/litmus/disk_fill/disk_fill_ds.j2
-        dest: /chaoslib/litmus/disk_fill/disk_fill_ds.yml
-      
-    - name: Create the DaemonSet for Disk-Fill
+        src:  /chaoslib/litmus/disk_fill/disk_fill_job.j2
+        dest: /chaoslib/litmus/disk_fill/disk_fill_job.yml
+      vars:
+        chaosUID: "{{ chaos_uid }}"
+        app_node: "{{ app_node }}"
+
+    - name: Create the Job for Disk-Fill
       shell: >
-        kubectl apply -f /chaoslib/litmus/disk_fill/disk_fill_ds.yml -n {{ a_ns }}
+        kubectl apply -f /chaoslib/litmus/disk_fill/disk_fill_job.yml -n {{ a_ns }}
       args:
         executable: /bin/bash
-      register: disk_fill_ds_result
+      register: disk_fill_job_result
   
-    - name: Confirm that the disk-fill daemonSet is running on all nodes
+    - name: Confirm that the disk-fill job is running on all nodes
       shell: >
         kubectl get pods -l app=disk-fill --no-headers --template "{{range .items}}{{if eq .spec.nodeName $(kubectl get nodes --no-headers --field-selector spec.unschedulable=false | grep Ready | awk '{printf "\"%s\" ", $1}')}}{{.status.phase}}{{\"\n\"}}{{end}}{{end}}" -n {{ a_ns }} | sort | uniq
```

Comment:
We are you adding a complex task to check the status of job. Can you please simply it?